protoc-gen-kotlin/lib/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     kotlin("multiplatform")
     `maven-publish`
+    id("com.google.osdetector") version "1.6.2"
```

Comment:
To make sure the Gradle plugin versions are consistent across the entire project, we declare all plugins in the top-level `build.gradle.kts` file with `apply false`. Then we apply them in the individual Gradle subprojects as needed.